app/pages/sections/catalogue-solutions/routes.js

```diff
@@ -45,8 +46,12 @@ export const catalogueSolutionsRoutes = (authProvider, addContext, sessionManage
   router.use('/select', catalogueSolutionsSelectRoutes(authProvider, addContext, sessionManager));
 
   router.get('/:orderItemId', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => {
-    const { orderId } = req.params;
+    const { orderId, orderItemId } = req.params;
     const accessToken = extractAccessToken({ req, tokenType: 'access' });
+
+    if (orderItemId) {
```

Comment:
we will always have an orderItemId. however we only want to call getOrderItem if the orderItemId is not `newsolution`